src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -523,10 +523,11 @@ public void MissingSatelliteAssemblyInLibrary ()
 		}
 
 		[Test]
-		public void MissingSatelliteAssemblyInApp ()
+		public void MissingSatelliteAssemblyInApp ([Values (false, true)] bool publishAot)
 		{
 			var proj = new XamarinAndroidApplicationProject {
 				IsRelease = true,
+				ProjectName = "MyApp",
```

Comment:
Is this change necessary?  Why?

Reply:
We could undo this line, I think it was when I was trying to get this right:
```csharp
var regex = $"--satellite:.+{proj.ProjectName}.resources.dll";
```